package.js

```diff
@@ -1,6 +1,6 @@
 Package.describe({
   name: 'okgrow:analytics',
-  version: '0.2.2',
+  version: '0.2.4',
```

Comment:
Bumped patch version (current version set on atmosphere is 0.2.3). Let me know if and how you want this changed.